runtime/sema/errors.go

```diff
@@ -615,7 +615,7 @@ type InvalidVariableKindError struct {
 
 func (e *InvalidVariableKindError) Error() string {
 	if e.Kind == ast.VariableKindNotSpecified {
-		return fmt.Sprintf("missing variable kind")
+		return "missing variable kind"
```

Comment:
this is a linter error fix. It creeped in from somewhere else.

Reply:
Thanks for fixing this